app/schema.json

```diff
@@ -209,14 +212,23 @@
               "children": { "type": "array" }
             }
           },
+          "created": { "type": "string" },
           "server": { "type": "string" },
           "tilesetUid": { "type": "string" },
           "position": { "type": "string" },
           "options": { "type": "object" },
           "name": { "type": "string" },
           "transforms": { "type": "array" },
           "width": { "type": "number" },
-          "height": { "type": "number" }
+          "height": { "type": "number" },
+          "binsPerDimension": { "type": "number" },
+          "orientation": { "type": "string" },
+          "local": { "type": "boolean" },
+          "thumbnail": { },
```

Comment:
This shouldn't be here either, I don't think. @flekschas do you know if the thumbnail should be in the viewconf output?